src/App.js

```diff
@@ -1,5 +1,50 @@
+import  Input  from './Input.js';
+import Lotto from './Lotto.js';
+import LottoResultCalculator from './LottoResultCalculator.js';
+import { validateBonusNum, validatePurchaseAmount, validateWinnigNum } from './validate.js';
+import { COMMON_MESSAGE } from './message.js';
+import { Console, MissionUtils } from "@woowacourse/mission-utils";
+
 class App {
-  async run() {}
+  async run() {
+    const inputPurchaseAmount = new Input(validatePurchaseAmount.validation);
+    await inputPurchaseAmount.inputValue(COMMON_MESSAGE.INPUT_PURCHASE);
+    const purchaseAmount = inputPurchaseAmount.getValue();
+    const lottoCount = inputPurchaseAmount.getLottoCount();
+
+    Console.print(`\n${lottoCount}개를 구매했습니다.`);
+
+    const lottos = [];
+    for(let i = 0; i < lottoCount; i++){
+      const numbers = this.generateLottoNumbers();
+      const lotto = new Lotto(numbers);
+      lottos.push(lotto);
+    }
```

Comment:
[13.6](https://github.com/ParkSB/javascript-style-guide?tab=readme-ov-file#variables--unary-increment-decrement) 이 부분을 보시면 ++ 단항 증감 연산자는 상황에 따라 예기치 못한 문제릴 일으킬 가능성이 있어서 지양하는 것이 좋다고 해요🥲 다음에 이것을 적용해보시면 좋을 것 같아요!~😄

Reply:
그리고 지금 보시면 새로운 배열을 만들기 위해 반복문을 사용하고 있어요! 이런 식으로 적을 수 있을 거예요!
```suggestion
    const lottos = Array.from({ length: lottoCount }, () => {
      const numbers = this.generateLottoNumbers();
      return new Lotto(numbers);
    });
```
이것은 Array의 메서드를 사용한 방식인데 처음에는 저도 아직 많이 어려워요🥲 from이라는 메서드에서 길이가 lottoCount인 배열을 만드는데, lottoCount 만큼 두 번째 인자에 있는 함수를 수행해요. 그리고 그 함수에서 반환하는 값을 각각 한 요소로 해서 배열을 완성하는 것이죠.
Array의 메서드들(from, map, forEach, reduce)는 자주 사용하기 때문에 한 번 시간을 들여서 다양한 예시를 보면서 익히는 것을 추천드려요! 다른 분들 PR들을 보는 것 만으로도 익숙해지기 좋을 것 같아요~ 그리고 그 이후에는 Object의 메서드들(values, keys, entries, fromEntries) 들을 공부하시면 아까 나온 메서드들과 연계해서 다양한 정보들을 조합해낼 수 있을 거예요!!☺️